client/app/components/composites/SideWinder/SideWinder.js

```diff
@@ -155,6 +156,7 @@ class SideWinder extends Component {
                 className: css.closeButton,
                 dangerouslySetInnerHTML: { __html: closeIcon },
               }),
+              span({ className: css.closeHelpText }, t('web.sidewinder.close')),
```

Comment:
We had an explicit conversation with @jannekoivistoinen earlier about not doing this. Sorry if you didn't know about this.

Did you test this on Mobile Safari? If you make a visual change when hovering, you need two taps for the click action to go through.